mockserver.js

```diff
@@ -131,9 +131,14 @@ var mockserver = {
         }
 
         var watchedHeaders = module.exports.headers;
+
         if(watchedHeaders && !Array.isArray(watchedHeaders)) {
             watchedHeaders = [watchedHeaders];
         }
+        if(!watchedHeaders || !watchedHeaders.length) {
+            watchedHeaders = ['authorization']
+        }
```

Comment:
Also should work :)

Kind Regards,
Nikolai Zujev

On Sun, Feb 7, 2016 at 12:47 PM, Alessandro Nadalin <
notifications@github.com> wrote:

> In mockserver.js
> <https://github.com/namshi/mockserver/pull/22#discussion_r52116242>:
>
> >          if(watchedHeaders && !Array.isArray(watchedHeaders)) {
> >              watchedHeaders = [watchedHeaders];
> >          }
> > +        if(!watchedHeaders || !watchedHeaders.length) {
> > +            watchedHeaders = ['authorization']
> > +        }
>
> What about using an env variable?
> … <#527375120_>
> On Feb 7, 2016 2:45 PM, "Nikolai Zujev" <notifications@github.com> wrote:
> In mockserver.js <#22 (comment)
> <https://github.com/namshi/mockserver/pull/22#discussion_r52115708>>: >
> if(watchedHeaders && !Array.isArray(watchedHeaders)) { > watchedHeaders =
> [watchedHeaders]; > } > + if(!watchedHeaders || !watchedHeaders.length) { >
> + watchedHeaders = ['authorization'] > + } Then I would suggest to migrate
> it to a CLI option, as I found myself to have custom header support, I had
> to reimplement bin file with overrides. That should come out of the box,
> imo — Reply to this email directly or view it on GitHub <
> https://github.com/namshi/mockserver/pull/22/files#r52115708>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/namshi/mockserver/pull/22/files#r52116242>.
>